test/integration/psetv2.spec.ts

```diff
@@ -366,15 +372,94 @@ describe('liquidjs-lib (transactions with psetv2)', () => {
       undefined,
       issuanceBlindingArgs,
     );
-    const blinder = new PsetBlinder(
+    let blinder = new PsetBlinder(
       pset,
       ownedInputs,
       zkpValidator,
       zkpGenerator,
     );
     await blinder.blindLast({ issuanceBlindingArgs, outputBlindingArgs });
-    const rawTx = signTransaction(pset, [alice.keys], Transaction.SIGHASH_ALL);
-    await regtestUtils.broadcast(rawTx.toHex());
+    const issuanceTx = signTransaction(
+      pset,
+      [alice.keys],
+      Transaction.SIGHASH_ALL,
+    );
+    const issuanceTxid = await regtestUtils.broadcast(issuanceTx.toHex());
+
+    const assetEntropy = issuanceEntropyFromInput(issuanceTx.ins[0]);
+    const reissuanceInputs = [
+      new CreatorInput(issuanceTxid, 0),
+      new CreatorInput(issuanceTxid, 3),
+    ];
+    const reissuanceOutputs = [
+      new CreatorOutput(
+        lbtc,
+        99998700,
+        alice.payment.output,
+        alice.payment.blindkey,
+        0,
+      ),
+      new CreatorOutput(lbtc, 700),
+    ];
+    const reissuancePset = PsetCreator.newPset({
+      inputs: reissuanceInputs,
+      outputs: reissuanceOutputs,
+    });
+
+    updater = new PsetUpdater(reissuancePset);
+    updater.addInWitnessUtxo(0, issuanceTx.outs[0]);
+    updater.addInWitnessUtxo(1, issuanceTx.outs[3]);
+    updater.addInUtxoRangeProof(0, issuanceTx.outs[0].rangeProof!);
+    updater.addInUtxoRangeProof(1, issuanceTx.outs[3].rangeProof!);
+    updater.addInReissuance(1, {
+      entropy: assetEntropy,
+      assetAmount: 1000,
+      assetAddress: alice.payment.confidentialAddress!,
+      tokenAmount: 1,
+      tokenAddress: alice.payment.confidentialAddress!,
+      tokenAssetBlinder: outputBlindingArgs[2].assetBlinder,
+    });
+    updater.addInSighashType(0, Transaction.SIGHASH_ALL);
+    updater.addInSighashType(1, Transaction.SIGHASH_ALL);
+
+    zkpGenerator = ZKPGenerator.fromInBlindingKeys([
+      alice.blindingKeys[0],
+      alice.blindingKeys[0],
+    ]);
+    const reissuanceownedInputs = await zkpGenerator.unblindInputs(
+      reissuancePset,
+    );
+    const reissuanceBlindingArgs = await zkpGenerator.blindIssuances(
+      reissuancePset,
+      {
+        1: alice.blindingKeys[0],
+      },
+    );
+
+    const reissuanceOutputBlindingArgs = await zkpGenerator.blindOutputs(
+      reissuancePset,
+      ZKPGenerator.ECCKeysGenerator(ecc),
+      undefined,
+      issuanceBlindingArgs,
```

Comment:
To make a normal re-issuance in a separated moment, how do we retrieve these issuanceBlindingArgs?

Reply:
You generate it via `await zkpGenerator.blindIssuances` call

Reply:
maybe the name is unfortunate, but ideally, if I am doing a reissuance, most likely the issuance has been done a long time ago by someone else?

Reply:
I don't actually understand the concern.
`blindIssuances` should be used only if one wants the input (re)issuance blinded while for the outputs this depends on the address, whether if it's confidential or not.
If we are all ok with the naming of `blindOutputs` that generates blinding data for the output to blind, then I don't `blindIssuances` is a bad name because it does the same thing, ie. generates blinding data for the input (re)issuances to blind.

> most likely the issuance has been done a long time ago by someone else?

I don't understand what this implies for you. The only one who can make a reissuance is the owner of the confidential reissuance token output because only he knows its asset blinder that will become the issuance asset blinding nonce, while the issuance entropy can be retrieved from blockchain.

Reply:
I mean, if I am spending a re-issuance token that is already blinded, do I need to do `blindIssuances` again? I suppose not?